src/persist.js

```diff
@@ -11,7 +12,15 @@ exports.filterLinks = async function (links) {
         postedOn: links.map(({ postedOn }) => postedOn)
       }
     })
-    const existing = new Set(pdfs.map((x) => x.postedOn))
+    /* Sequelize erroneously adds timezone to DATEONLY on retrieval from MSSQL
+    so we need to add a day to make it right */
+    const existing = new Set(pdfs.map((x) => {
```

Comment:
1. Only proceed to 2. when `if postedOn.time is not midnight`
2. Load postedOn into a Moment object
3. then get the timezone offset where the client is running (using Moment if possible, Date otherwise)
4. then add that to the postedOn Moment object